backend/src/controllers/authentication.spec.ts

```diff
@@ -10,38 +9,16 @@ describe('Should test login', () =>
 
 	it('Should get token from login', async() =>
 	{
-		const response : Response = await supertest(server).post(environment.apiRoutes.register).send(
-		{
-			email : 'jestForLogin.email@mail.com',
-			password : '123456'
-		});
-		const body : User = await response.body;
-
-		expect(response.statusCode).toBe(200);
-		expect(response.headers).toBeDefined();
-		expect(body.email).toContain('jestForLogin.email@mail.com');
-		expect(body.password).not.toHaveLength(0);
-
 		const loginResponse : Response = await supertest(server).post(environment.apiRoutes.login).send(
 		{
-			email : 'jestForLogin.email@mail.com',
-			password : '123456'
+			email : 'test@test.com',
```

Comment:
What about a integration test that does have no challenge? You could at least check for the status code.